packages/backup-restore-operator/charts/templates/rancher-resourceset.yaml

```diff
@@ -17,16 +17,32 @@ resourceSelectors:
   - apiVersion: "rbac.authorization.k8s.io/v1"
     kindsRegexp: "^clusterrolebindings$"
     resourceNameRegexp: "^cattle-|^clusterrolebinding-|^globaladmin-user-|^grb-u-"
+    resourceNames:
+      - "eks-operator"
   - apiVersion: "rbac.authorization.k8s.io/v1"
     kindsRegexp: "^clusterroles$"
     resourceNameRegexp: "^cattle-|^p-|^c-|^local-|^user-|^u-|^project-|^create-ns$"
+    resourceNames:
+      - "eks-operator"
   - apiVersion: "apiextensions.k8s.io/v1beta1"
     kindsRegexp: "."
-    resourceNameRegexp: "management.cattle.io$|project.cattle.io$"
+    resourceNameRegexp: "management.cattle.io$|project.cattle.io$|catalog.cattle.io$|eks.cattle.io$|resources.cattle.io$"
```

Comment:
How are we handling 2.5 CRDs such as Monitoring, Istio, Logging etc. that are in different resource groups like `monitoring.coreos.com`? Will feature charts be expected to create a resourceset on chart deploy similar to how monitoring expects feature charts to create ServiceMonitors?

Reply:
yes we will have to create ResourceSets per 2.5 feature chart.